commons/commons-util/src/main/kotlin/ebi/ac/uk/base/StringExt.kt

```diff
@@ -67,3 +69,8 @@ fun String.scape(toScape: String, scapeLiteral: String = "\\"): String = replace
  * Replaces multiple whitespaces with a single space.
  */
 fun String.trim(): String = replace("\\s+".toRegex(), " ")
+
+/**
+ * Compute the given string so it always ends in the provided suffix, no action is taking if string already.
```

Comment:
I'd suggest:

... provided suffix. No action will be taken if the string already finishes with the expected suffix.